test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -1035,3 +1037,65 @@
     (is (= [{:value_type_id 4
               :value nil}]
            (jdbc/query-to-vec "select value_type_id, value from fact_values")))))
+
+(deftest migration-67-adds-hashes-to-resource-events
+  (let [current-time (to-timestamp (now))]
+    (jdbc/with-db-connection *db*
+      (clear-db-for-testing!)
+      (fast-forward-to-migration! 66)
+
+      (jdbc/insert! :report_statuses
+                    {:status "testing1" :id 1})
+      (jdbc/insert! :environments {:id 0 :environment "testing"})
+      (jdbc/insert! :certnames {:certname "a.com"})
+
```

Comment:
Not sure, but wondered if we might be able to use `puppetlabs.puppetdb.examples.reports` here (could look at the other uses via `git grep -F examples.reports`).

Reply:
Okay. I followed the pattern of another migration test already in use, but changing this is trivial.

Reply:
Oh, well it might be that what you have is what we want -- just wondered.